person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, name, age = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] The default value "unknown" should be assigned to `name`, not `age`.

- [ ] Let's also change the `id` so that it is random and we won't need to add it everytime we create a person.

  ```
  def initialize(age, name = "Unknown", parent_permission: true)
    @id = rand(1...1000)
    @age = age
    @name = name
    @parent_permission
  end
  ```